packages/server/lib/cypress.js

```diff
@@ -59,6 +59,10 @@ const exitErr = (err) => {
   })
 }
 
+const isComponentTesting = (options) => {
+  return options.testingType === 'component'
```

Comment:
Do you think we should have constants for those values? I feel like "component" is a pretty common word that would be hard to search and replace come refactoring day.

Reply:
let's just TS all the things!

Reply:
I looked into this but TSing all the things is a lot of work (cypress.js and surrounding files are big).

As for exporting constants, this pattern doesn't seem very common in the code-base - I think rather than introducing a new abstraction, we should focus on TS-ing all the thing, which solves this problem. It's a bit more work than makes sense to do here, though... but 100% agree with you.